<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>509</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bt5">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>47</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Away (Y-)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt6">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>105</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Towards (Y+)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Left (X+)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>93</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Right (X-)</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>241</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Distance/click:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(mm)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Position</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt8">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>93</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Zero</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>51</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Down (Z+)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>51</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Up (Z-)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>231</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Map Straight Line</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QSpinBox" name="NumPoints">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string># of Sample Points</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_1">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Z_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Position [x,y,z]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Z_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>100</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Position [x,y,z]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Line">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="X_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="X_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>200</y>
      <width>231</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Map Cylinder</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QSpinBox" name="Z_Points">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string># of Sample Points [z]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Z_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>150</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Position [x,y,z]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Z_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>180</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>End Position [x,y,z]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Cylinder">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>210</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Phi_Points">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string># of Sample Points [phi]</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="Radius">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius [mm]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="X_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="X_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Map Box</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string># of Sample Points [y]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string># of Sample Points [z]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Z_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Box">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>200</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="x_points">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="y_points">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>End Position [x,y,z]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="Y_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Z_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Position [x,y,z]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string># of Sample Points [x]</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="z_points">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="X_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="X_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>280</y>
      <width>231</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Map Sphere</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Note: Start point should be zero'd at magnetic center!</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>62</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Radius:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="Radius_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>60</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>450</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>55</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>[mm]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Sphere">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>812</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRoboRunner">
    <property name="title">
     <string>RoboRunner</string>
    </property>
   </widget>
   <addaction name="menuRoboRunner"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>bt1</tabstop>
  <tabstop>bt2</tabstop>
  <tabstop>bt3</tabstop>
  <tabstop>bt4</tabstop>
  <tabstop>bt5</tabstop>
  <tabstop>bt6</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>bt7</tabstop>
  <tabstop>bt8</tabstop>
  <tabstop>NumPoints</tabstop>
  <tabstop>X_1</tabstop>
  <tabstop>Y_1</tabstop>
  <tabstop>Z_1</tabstop>
  <tabstop>X_2</tabstop>
  <tabstop>Y_2</tabstop>
  <tabstop>Z_2</tabstop>
  <tabstop>Line</tabstop>
  <tabstop>Z_Points</tabstop>
  <tabstop>Phi_Points</tabstop>
  <tabstop>Radius</tabstop>
  <tabstop>X_3</tabstop>
  <tabstop>Y_3</tabstop>
  <tabstop>Z_3</tabstop>
  <tabstop>X_4</tabstop>
  <tabstop>Y_4</tabstop>
  <tabstop>Z_4</tabstop>
  <tabstop>Cylinder</tabstop>
  <tabstop>x_points</tabstop>
  <tabstop>y_points</tabstop>
  <tabstop>z_points</tabstop>
  <tabstop>X_5</tabstop>
  <tabstop>Y_5</tabstop>
  <tabstop>Z_5</tabstop>
  <tabstop>X_6</tabstop>
  <tabstop>Y_6</tabstop>
  <tabstop>Z_6</tabstop>
  <tabstop>Box</tabstop>
  <tabstop>Exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
